pom.xml

```diff
@@ -51,8 +51,8 @@
         <slf4j.version>1.7.25</slf4j.version>
         <slf4j-log4j12.version>1.7.25</slf4j-log4j12.version>
         <guava.version>19.0</guava.version>
-        <netty.version>4.1.75.Final</netty.version>
-        <netty-tcnative.version>2.0.50.Final</netty-tcnative.version>
+        <netty.version>4.1.77.Final</netty.version>
+        <netty-tcnative.version>2.0.52.Final</netty-tcnative.version>
```

Comment:
As discussed in prior updates version here is taken from [the base pom in Netty](https://github.com/netty/netty/blob/netty-4.1.77.Final/pom.xml#L531)